src/model/openthaigpt_pretraining_model/models/nanoGPT/model.py

```diff
@@ -28,8 +59,490 @@ def _attn_wrapper(self, query, key, value, attention_mask=None, head_mask=None):
     return attn_out, None
 
 
+class EditGPT2Attention(OriginalGPT2Attention):
```

Comment:
I think `GPT2AttentionWithRotary` is better name.

Reply:
DONE

Reply:
You should changed all of class name in your code.

Reply:
DONE